code/modules/jobs/jobs.dm

```diff
@@ -56,6 +56,12 @@ GLOBAL_LIST_INIT(security_positions, list(
 	"Detective",
 	"Security Officer"))
 
+/// These aren't defacto jobs, but are the special departmental variants for sec officers.
+GLOBAL_LIST_INIT(security_sub_positions, list(
+	"Security Officer (Cargo)",
+	"Security Officer (Engineering)",
+	"Security Officer (Medical)",
+	"Security Officer (Science)",))
```

Comment:
```suggestion
	"Security Officer (Science)",
))
```

Reply:
To be clear, the whole point of trailing commas is that they preserve blames (and also for consistency). Thus, the parentheses must be on their own line.